src/modules/map/components/cesium/CesiumMouseTracker.vue

```diff
@@ -0,0 +1,121 @@
+<script setup>
+import { Cartographic, Math, ScreenSpaceEventHandler, ScreenSpaceEventType } from 'cesium'
+import { transform } from 'ol/proj'
+import { computed, inject, nextTick, onMounted, onUnmounted, ref, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import allFormats, { LV03Format, LV95Format } from '@/utils/coordinates/coordinateFormat'
+import log from '@/utils/logging'
+
+const mousePosition = ref(null)
+const displayedFormatId = ref(LV95Format.id)
+
+const store = useStore()
+const i18n = useI18n()
+
+const is3DReady = computed(() => store.state.cesium.isViewerReady)
+const projection = computed(() => store.state.position.projection)
+
+const dispatcher = { dispatcher: 'CesiumMouseTracker.vue' }
+const getViewer = inject('getViewer', () => {}, true)
+
+let handler = null
+
+watch(
+    is3DReady,
+    (newValue) => {
+        if (newValue) {
+            setupHandler()
+        }
+    },
+    { immediate: true }
+)
+
+onMounted(() => {
+    nextTick(() => {
+        setupHandler()
+    })
+})
+
+onUnmounted(() => {
+    if (handler) {
+        handler.removeInputAction(ScreenSpaceEventType.MOUSE_MOVE)
+    }
+})
+
+function setupHandler() {
+    if (!getViewer()) {
+        return
+    }
+    handler = new ScreenSpaceEventHandler(getViewer().scene.canvas)
+    const viewer = getViewer()
+    handler.setInputAction((movement) => {
+        const ray = viewer.camera.getPickRay(movement.endPosition)
+        const cartesian = viewer.scene.globe.pick(ray, viewer.scene)
+        if (cartesian) {
+            const cartographic = Cartographic.fromCartesian(cartesian)
+            const longitude = Math.toDegrees(cartographic.longitude)
+            const latitude = Math.toDegrees(cartographic.latitude)
+            let coordinate = transform([longitude, latitude], 'EPSG:4326', projection.value)
```

Comment:
I'd prefer you use `proj4` here, instead of relying on OL utils in Cesium context